alf/bin/play.py

```diff
@@ -121,6 +121,7 @@ def play():
     else:
         alf.config('create_environment', for_evaluation=True, nonparallel=True)
     alf.config('TrainerConfig', mutable=False, random_seed=seed)
+    common.info('Playing with random seed: {}'.format(seed))
```

Comment:
However, if the seed is not explicitly specified when calling common.set_random_seed() at line 112, torch deterministic mode  will not be enabled. So even supplying the same seed later may not reproduce earlier result.

Reply:
True. This is not the best way to use random seed for reproducing previous results. Close it for now.